src/com/company/Calculator/CalculatorGui.java

```diff
@@ -0,0 +1,78 @@
+package com.company.Calculator;
+
+import com.company.Config.ConfigLoader;
+import com.company.Pair;
+
+import java.util.*;
+
+import static com.company.Calculator.CalculatorLogic.*;
+import static com.company.Constants.*;
+
+public class CalculatorGui {
+
+    public static final String CONFIG_PATH = "src\\com\\company\\config\\ValidOperators.properties";
+    private static final String NEW_LINE = "\n";
+    private static final ArrayList<String> VALID_OPERATORS = ConfigLoader.loadConfig(CONFIG_PATH);
+
+    public static void menu() {
```

Comment:
![image](https://user-images.githubusercontent.com/17217900/79755584-08eb8180-8322-11ea-9961-4850a8de03a4.png)

Reply:
Why are you not checking if the equation is ok? I used the valid operators, what is wrong? fix it.